source/includes/_configuration-and-environments.md

```diff
@@ -0,0 +1,81 @@
+# Configuration and environments
+One of the goals of booster is to avoid configuration parameters as much as
+possible, inferring everything automatically from your code. However, there
+are some aspects that can't be inferred (like the application name), or it
+is useful to change them under certain circumstances, especially when using
+different [environments](#environments)
+
+## Booster configuration
+```typescript
+import { Booster } from '@boostercloud/framework-core'
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { Provider } from '@boostercloud/framework-provider-aws'
+
+Booster.configure('pre-production', (config: BoosterConfig): void => {
+  config.appName = 'my-app-name'
+  config.provider = Provider
+})
+```
+You configure your Booster application by calling the `Booster.configure()` method.
+Typically, this call is done in a file placed inside "src/config" folder. If you have
+used the `boost new:project` generator, you will already have this file there.
+
+The following is the list of the fields you can configure:
+
+- **appName:** This is the name that identifies your application. It will be used
+for many things, such us prefixing the resources created by the provider. There
+are certain restrictions regarding the characters you can use: all of them must be
+lower-cased and can't contain spaces.
+Two apps with different names are completely independent.
+- **provider:** This field contains the provider you will be using when deploying
+or running your application. So far, there is only one provider created for Booster,
+`@boostercloud/framework-provider-aws`, and it is probably the one you have already
+set if you used the generator to create your project. There will be more providers,
+like one that allows running your Booster app locally.
+
+## Environments
+
```

Comment:
```suggestion

You can configure multiple environments calling to the `Booster.configure` function several times using different environment names:

```

Reply:
Take into account that the reason why the code-blocks are first is because Slate, so people reading the docs will see the text first and then, on the right, they will see the code ;-)